backoffice/backoffice/workflows/tests/cassettes/TestAirflowUtils.test_find_failed_dag.yaml

```diff
@@ -0,0 +1,184 @@
+  response:
+    body:
+      string: "{\n  \"conf\": {\n    \"workflow_id\": \"00000000-0000-0000-0000-000000000001\"\n
+        \ },\n  \"dag_id\": \"author_create_initialization_dag\",\n  \"dag_run_id\":
+        \"00000000-0000-0000-0000-000000000001\",\n  \"data_interval_end\": \"2024-07-30T08:29:55.039650+00:00\",\n
+        \ \"data_interval_start\": \"2024-07-30T08:29:55.039650+00:00\",\n  \"end_date\":
+        null,\n  \"execution_date\": \"2024-07-30T08:29:55.039650+00:00\",\n  \"external_trigger\":
+        true,\n  \"last_scheduling_decision\": null,\n  \"logical_date\": \"2024-07-30T08:29:55.039650+00:00\",\n
+        \ \"note\": null,\n  \"run_type\": \"manual\",\n  \"start_date\": null,\n
+        \ \"state\": \"queued\"\n}\n"
+    headers:
+      Connection:
+      - close
+      Content-Length:
+      - '579'
+      Content-Type:
+      - application/json
+      Date:
+      - Tue, 30 Jul 2024 08:29:55 GMT
+      Server:
+      - gunicorn
+      X-Robots-Tag:
+      - noindex, nofollow
+    status:
+      code: 200
+      message: OK
+- request:
+    body: null
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Type:
+      - application/json
+    method: GET
+    uri: http://airflow-webserver:8080/api/v1/dags/author_create_initialization_dag/dagRuns/00000000-0000-0000-0000-000000000001
+  response:
+    body:
+      string: "{\n  \"conf\": {\n    \"workflow_id\": \"00000000-0000-0000-0000-000000000001\"\n
+        \ },\n  \"dag_id\": \"author_create_initialization_dag\",\n  \"dag_run_id\":
+        \"00000000-0000-0000-0000-000000000001\",\n  \"data_interval_end\": \"2024-07-30T08:29:55.039650+00:00\",\n
+        \ \"data_interval_start\": \"2024-07-30T08:29:55.039650+00:00\",\n  \"end_date\":
+        \"2024-07-30T08:30:12.831165+00:00\",\n  \"execution_date\": \"2024-07-30T08:29:55.039650+00:00\",\n
+        \ \"external_trigger\": true,\n  \"last_scheduling_decision\": \"2024-07-30T08:30:12.828982+00:00\",\n
+        \ \"logical_date\": \"2024-07-30T08:29:55.039650+00:00\",\n  \"note\": null,\n
+        \ \"run_type\": \"manual\",\n  \"start_date\": \"2024-07-30T08:29:55.206573+00:00\",\n
+        \ \"state\": \"failed\"\n}\n"
+    headers:
+      Connection:
+      - close
+      Content-Length:
+      - '669'
+      Content-Type:
+      - application/json
+      Date:
+      - Tue, 30 Jul 2024 08:30:15 GMT
+      Server:
+      - gunicorn
+      X-Robots-Tag:
+      - noindex, nofollow
+    status:
+      code: 200
+      message: OK
+- request:
+    body: null
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Type:
+      - application/json
+    method: GET
+    uri: http://airflow-webserver:8080/api/v1/dags/author_create_approved_dag/dagRuns/00000000-0000-0000-0000-000000000001
```

Comment:
there is something wrong, it returns 404 actually. Could we please double check all the cassettes that we have the correct responses?